lib/active_admin/filters/active_filter.rb

```diff
@@ -26,13 +26,21 @@ def values
       end
 
       def label
+        translated_predicate = predicate_name.mb_chars.downcase.to_s
         if related_class
-          "#{related_class.model_name.human} #{predicate_name}".strip
+          "#{related_class.model_name.human} #{translated_predicate}".strip
         else
-          "#{attribute_name} #{predicate_name}".strip
+          "#{attribute_name} #{translated_predicate}".strip
         end
       end
 
+      # translated active admins predicated equals, contains, starts_with ..
+      # with fallback to ransack eq, cont, start ...
+      def predicate_name
+        key = condition.predicate.name
+        I18n.t("active_admin.filters.predicates.#{key}", default: Ransack::Translate.predicate(key))
+      end
```

Comment:
```
def fallback_to_ransack_predicate_name
  Ransack::Translate.predicate(condition.predicate.name)
end
```

Now we are back to the original method, just named differently and in another position in the file. I think the old name is good enough, `predicate_name` (vs the new `translated_predicate_name`).